src/components/editor/Bindings/Editor.tsx

```diff
@@ -141,19 +146,14 @@ function BindingsEditor({ loading, skeleton, readOnly = false }: Props) {
 
                             {collectionData ? (
                                 collectionData.belongsToDraft ? (
-                                    <DraftSpecEditor
-                                        entityType="collection"
-                                        localZustandScope={true}
-                                        editorHeight={EDITOR_HEIGHT}
+                                    <CollectionSchemaEditor
                                         entityName={currentCollection}
                                     />
                                 ) : (
                                     <ControlledEditor />
                                 )
                             ) : (
-                                <MonacoEditorSkeleton
-                                    editorHeight={EDITOR_HEIGHT}
-                                />
+                                <CollectionSchemaEditorSkeleton />
```

Comment:
This is a bit wonky but I feel like in most cases the user will land on the new editor... so going with that skeleton for now.

This is a funny edge case making we thinking about how we handle these skeletons.